common/cudautils/include/cudautils/cudautils.hpp

```diff
@@ -26,9 +26,19 @@
         claragenomics::cudautils::gpuAssert((ans), __FILE__, __LINE__); \
     }
 
+/// \def CGA_CUDA_BEFORE_XX_X
+/// \brief Macros to enable/disable CUDA version specific code
+#define CGA_CUDA_BEFORE_XX_X 1
+
+#if __CUDACC_VER_MAJOR__ < 10 || (__CUDACC_VER_MAJOR__ == 10 && __CUDACC_VER_MINOR__ < 1)
+#define CGA_CUDA_BEFORE_10_1 1
+#else
+#define CGA_CUDA_BEFORE_10_1 0
+#endif
+
 /// \def CGA_CONSTEXPR
 /// \brief C++ constexpr for device code - falls back to const for CUDA 10.0 and earlier
-#if __CUDACC_VER_MAJOR__ < 10 || (__CUDACC_VER_MAJOR__ == 10 && __CUDACC_VER_MINOR__ < 1)
+#if CGA_CUDA_BEFORE_10_1
 #define CGA_CONSTEXPR const
 #else
 #define CGA_CONSTEXPR constexpr
```

Comment:
won't this `#define CGA_CONSTEXPR constexpr` for CUDA=10.1, which causes the nvcc error?

Reply:
Yes, but `constexpr` should not be a problem in 10.1.